src/.vuepress/public/editor.html

```diff
@@ -647,6 +641,19 @@ <h5>Documentation</h5>
         if (!evt.target.checked) optionZeroFilledMemory.checked = false;
       })
 
+      // WebAssembly GC depends on Reference Types
+      let optionFeatureReferencetypesEnabled = optionFeatureReferencetypes.checked;
+      optionFeatureWasmGC.addEventListener('change', evt => {
+        if (evt.target.checked) {
+          optionFeatureReferencetypesEnabled = optionFeatureReferencetypes.checked;
+          optionFeatureReferencetypes.checked = true;
+          optionFeatureReferencetypes.disabled = true;
+        } else {
+          optionFeatureReferencetypes.disabled = false;
+          optionFeatureReferencetypes.checked = optionFeatureReferencetypesEnabled;
+        }
+      })
+
```

Comment:
Wondering if it would be simpler, when enabling GC, that reference types is automatically enabled as well, respectively when disabling reference types, GC is also disabled automatically? Just an idea :)

Reply:
Ah, I see it currently does this already, modulo disabling GC when RT is disabled, since RT cannot be disabled when GC is enabled.